src/core/config.go

```diff
@@ -525,6 +526,7 @@ type Configuration struct {
 		ProtocGoPlugin   string   `help:"The binary passed to protoc as a plugin to generate Go code. Defaults to protoc-gen-go.\nWe've found this easier to manage with a go_get rule instead though, so you can also pass a build label here. See the Please repo for an example." var:"PROTOC_GO_PLUGIN"`
 		GrpcPythonPlugin string   `help:"The plugin invoked to compile Python code for grpc_library.\nDefaults to protoc-gen-grpc-python." var:"GRPC_PYTHON_PLUGIN"`
 		GrpcJavaPlugin   string   `help:"The plugin invoked to compile Java code for grpc_library.\nDefaults to protoc-gen-grpc-java." var:"GRPC_JAVA_PLUGIN"`
+		GrpcGoPlugin     string   `help:"The plugin invoked to compile Go code for grpc_library.\nDefaults to protoc-gen-go-grpc." var:"GRPC_GO_PLUGIN"`
```

Comment:
Should we skip the feature flag and just operate on this config value? Default it to empty, but use it if it's set?

Reply:
Yeah I guess I wanted all the config options here to behave the same (i.e. look for the plugin on the path) but go seems to behave differently to the other languages anyway.